### ISSUE TITLE
Making the overflow menu translucent

### ISSUE THREAD

**Issue by Tomikbobi** [ramensoftware/windows-11-taskbar-styling-guide#83]
iam trying to find a way to make the overflow menu translucent like the notifications trough the notification center styler

**Comment by bbmaster123** [comment]
```
Target: Border#OverflowFlyoutBackgroundBorder
Style: Background=Transparent
```

**Comment by bbmaster123** [comment]
make sure transparency effects are turned on in settings under personalization > colors
also, there is a known issue with transparency if you have [multiple monitors](https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/80)

**Comment by Tomikbobi** [comment]
Already have the setting enabled and iam using only one monitor thats why i cant figure out the issue

**Comment by bbmaster123** [comment]
do you have any other mods enabled that affect the taskbar? for example TaskbarX, Startallback, etc
could you send a log? logging can be turned on in advanced

**Comment by Tomikbobi** [comment]
I have translucentTB and thats about it. I will send log later then

**Comment by m417z** [comment]
There's a known incompatibility with TranslucentTB, see https://github.com/ramensoftware/windows-11-taskbar-styling-guide/issues/9#issuecomment-1896063781

**Comment by Tomikbobi** [comment]
still even when i turned off translucenttb and restarted it still wont work, but when i try just Background=transparent as style then it is workin but it doesnt look like i wanted

**Comment by Tomikbobi** [comment]
![Snímek obrazovky 2024-08-28 202743](https://github.com/user-attachments/assets/892765b6-6995-46f5-a7a7-1bb23c2b86d6)
i hope that i did it right

**Comment by bbmaster123** [comment]
almost, but it would help if I could see the whole thing.  Please do the following:

1. Disable the mod
2. Turn on logging
3. Enable the mod
4. Copy/paste everything into notepad, save as a .txt
5. drag and drop the txt file here

**Comment by Tomikbobi** [comment]
[log.txt](https://github.com/user-attachments/files/16788257/log.txt)
like this?

**Comment by bbmaster123** [comment]
ok so the last part of the log stands out to me:
```
00000058	20.21943092	[4136] [WH] LL	
00000059	21.22572517	[5404] [WH] LL	
00000060	21.24013901	[5404] [WH] GPA	
00000061	21.24369240	[5404] [WH] II	
00000062	21.24517059	[5404] [WH] [InjectInit]: Running InjectInit	
```

it ends there, but it should continue on, even with no styles added. 

Based on what I see, I think it might not be hooking explorer correctly, even though it partially works. @m417z might know more about what could be going on here, but in the meantime here's a bunch of things:

1. do you have a forked version of the mod which is also enabled?
2. did you add anything to the include/exclude lists?
3. did you replace explorer.exe or any dll's with another version?
4. can you please verify there is no other customization outside of Windhawk and TranslucentTB that would affect the taskbar? Please double check startup apps and task scheduler, and if anything does show up, temporarily disable them
5. is there anything else unique to your configuration that you can think of? 
6. can you verify that your system image is healthy using the dism commands followed by `sfc /scannow`?
7. have you tried uninstalling/reinstalling windhawk? is windhawk running as admin?
8. did you change any vivetool ID's?

**Comment by Tomikbobi** [comment]
checked each thing and i dont think so, but i think i finally figured it out (kinda). The problem is apparantly with the AcrylicBrush. I cant believe that i didnt tried any other color or style to this point. So now why wouldnt AcrylicBrush work xd

**Comment by bbmaster123** [comment]
Can you try SolidColorBrush  or LinearGradientBrush? is it a problem with:
a. xaml brushes
b. transparency

let me know

**Comment by Tomikbobi** [comment]
both of them works as well as transparent

**Comment by Tomikbobi** [comment]
weird thing is that the acrylicbrush works in the other mod (Windows 11 Notification Center Styler)

**Comment by bbmaster123** [comment]
then I'm not sure, AcrylicBrush should work fine if other brushes are working and it works in notification center

that being said, your log mentioned you were on 22621, you could try updating to 22631, otherwise I'd think something is conflicting or something about windows is a bit corrupt. 

what does it say when you run dism and sfc?

1. open cmd as admin
2. dism /online /cleanup-image checkhealth - what does it say?
3. dism /online /cleanup-image scanhealth
4. dism /online /cleanup-image restorehealth - what does it say?
5. reboot
6. open cmd as admin
7. sfc /scannow - what does this say?

**Comment by Tomikbobi** [comment]
For the last 20mins or so iam stuck updating no progress bar nothing

**Comment by Tomikbobi** [comment]
2.
Deployment Image Servicing and Management tool
Version: 10.0.22621.2792

Image Version: 10.0.22621.4112

No component store corruption detected.
The operation completed successfully.
4.
Deployment Image Servicing and Management tool
Version: 10.0.22621.2792

Image Version: 10.0.22621.4112

[==========================100.0%==========================] The restore operation completed successfully.
The operation completed successfully.

**Comment by m417z** [comment]
Looks similar to this report:
https://github.com/ramensoftware/windhawk-mods/issues/883

I don't know why the diagnostics don't fire up the element enumeration. Try to find a way to reproduce it so that I'll be able to debug it and investigate the problem.

It's also surprising that having it transparent works. That doesn't make sense to me, I'd expect the mod to not work at all if element enumeration doesn't work. If you can post logs with both trying the transparent style and trying the translucent style, it might provide some insights.

**Comment by Tomikbobi** [comment]
[traslucent.txt](https://github.com/user-attachments/files/16798081/traslucent.txt)
[transparent.txt](https://github.com/user-attachments/files/16798083/transparent.txt)
hope that i did it right

**Comment by m417z** [comment]
I see that the reason logs are missing is that you enabled Windhawk logging, but didn't enable mod logging.

Attach debug logs by following the instructions on the Troubleshooting page:
https://github.com/ramensoftware/windhawk/wiki/Troubleshooting#some-or-all-windhawk-mods-dont-work

> * Disable the mod.
> * Go to the mod's Advanced tab.
> * Set **Debug logging** to **Detailed debug logs**.
> * Click on **Show log output**.
> * Enable the mod.
> * Verify that the mod doesn't work, copy the log and save it to a file.
> * Set **Debug logging** back to **None**.

**Comment by Tomikbobi** [comment]
[debug.txt](https://github.com/user-attachments/files/16818305/debug.txt)
like this?

**Comment by m417z** [comment]
Yes, I see that it's `<AcrylicBrush TintColor="Transparent" />` is configured, but it wasn't used during the logging. Did you open the overflow tray during logging?

**Comment by Tomikbobi** [comment]
i dont think so
